### PR TITLE
tests: use lvm batch on osd2 (all_daemons)

### PR DESCRIPTION
**Pull request by guits**
in order to test lvm batch in purge scenario.

Signed-off-by: Guillaume Abrioux <gabrioux@redhat.com>

### CODE REVIEW

**Comment by guits** [comment]
jenkins test centos-non_container-purge

**Comment by guits** [comment]
jenkins test centos-container-purge

**Comment by guits** [comment]
jenkins test centos-container-purge

**Comment by guits** [comment]
jenkins test centos-non_container-purge

**Comment by guits** [comment]
jenkins test centos-non_container-purge

**Comment by guits** [comment]
jenkins test centos-container-purge

File: tox.ini [hunk 1]
```diff
@@ -337,8 +337,8 @@ commands=
   bash {toxinidir}/tests/scripts/vagrant_up.sh --no-provision {posargs:--provider=virtualbox}
   bash {toxinidir}/tests/scripts/generate_ssh_config.sh {changedir}
 
-  # configure lvm
-  !lvm_batch-!lvm_auto_discovery: ansible-playbook -vv -i {changedir}/{env:INVENTORY} {toxinidir}/tests/functional/lvm_setup.yml
+  # configure lvm, we exclude osd2 given this node uses lvm batch scenario (see corresponding inventory host file)
+  !lvm_batch-!lvm_auto_discovery: ansible-playbook -vv -i {changedir}/{env:INVENTORY} {toxinidir}/tests/functional/lvm_setup.yml --limit 'osds:!osd2'
```
Comment (dsavineau) [inline tox.ini:341]: we need this in the `[purge]` section too

https://github.com/ceph/ceph-ansible/blob/master/tox.ini#L60

**Review by dsavineau — CHANGES_REQUESTED** [review]

**Comment by guits** [comment]
jenkins test centos-container-purge

**Comment by guits** [comment]
jenkins test centos-non_container-purge

**Comment by guits** [comment]
jenkins test centos-non_container-purge

**Comment by guits** [comment]
jenkins test centos-non_container-all_daemons

**Comment by guits** [comment]
jenkins test centos-container-all_daemons

**Review by dsavineau — APPROVED** [review]

**Pull request merged**